packages/nextra/src/loader.ts

```diff
@@ -109,7 +115,10 @@ async function getPageMap(currentResourcePath: string): Promise<PageMapResult> {
       .map(item => {
         if (!item) return
         if (item.route === activeRoute) {
-          activeRouteTitle = dirMeta[item.name] || item.name
+          const metadata = dirMeta[item.name]
+          activeRouteTitle =
+            (typeof metadata === 'string' ? metadata : metadata.title) ||
```

Comment:
The dirMeta could be `{}` in this suitation. So reading ```metadata.title``` is not safe here.

![error.png](https://vip1.loli.io/2022/01/09/tDsObKAVnu9THPQ.png)